### ISSUE TITLE
ter-newline-after-var shows unexpected error

### ISSUE THREAD

**Issue by webschik** [buzinas/tslint-eslint-rules#302]
In the next code, the validator shows an error 'Expected blank line after variable declarations.', but it shouldn't. 

```
const dateOfBirth: string = value.slice(1, 9); // yyyymmdd

value = '...';
```
